boost-common/src/main/java/io/openliberty/boost/common/utils/LibertyBoosterUtil.java

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.boost.common.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import io.openliberty.boost.common.BoostLoggerI;
+import io.openliberty.boost.common.config.BoosterPackConfigurator;
+import io.openliberty.boost.common.config.JAXRSBoosterPackConfigurator;
+import io.openliberty.boost.common.config.JDBCBoosterPackConfigurator;
+import io.openliberty.boost.common.config.LibertyServerConfigGenerator;
+
+public class LibertyBoosterUtil {
```

Comment:
I think we should change this class to use static methods like the SpringBootUtil and the other helper classes.

Reply:
I kind of agree, not just for the sake of consistency but also because it steers us to thinking more carefully about object interactions and lifecycles to give more specific names to things we're creating and holding an instance of... vs. code that just statelessly returns output from input.    If it's really something we want an instance of... we should be able to find a better name than xxxUtil

Reply:
I think having an instantiation of this class will help reduce the number of times we need to pass the same variables to the methods. It also pushes the resolution of the BoosterConfigurators into common rather than having the gradle and maven plugins doing that. Other than consistency with other helper classes, is there a preference to having static classes?

Reply:
Hey @scottkurz  - Sorry, I commented before seeing your comment. Either that or git is putting out chain out of order.  Which way do you agree with?

Reply:
I tend to agree that they should be static. Especially since they are in common and are basically helper methods.

Reply:
That's fine with me. @mattbsox - I can make this change in the next issue if you are ok with that.  Just to move this issue along for now.